backends/jwt_javascript.go

```diff
@@ -90,14 +96,21 @@ func (o *jsJWTChecker) GetUser(token string) (bool, error) {
 	}
 
 	if o.options.parseToken {
-		username, err := getUsernameForToken(o.options, token, o.options.skipUserExpiration)
+		claims, err := getClaimsForToken(o.options, token, o.options.skipUserExpiration)
 
 		if err != nil {
-			log.Printf("jwt get user error: %s", err)
+			log.Printf("jwt get claims error: %s", err)
 			return false, err
 		}
 
-		params["username"] = username
+		if o.passClaims {
+			params["claims"] = claims
+		}
```

Comment:
[Nitpick] please add an empty line between the conditionals.

Reply:
Done